File: cs50-2020/pset1/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int i = 0;
    int k = 0;
    int j = 0;
    int h = 0;
    int n = 0;

    do
    {
        h = get_int("Height: ");

    }    
    while (h < 1 || h > 8);
    n = h;
    for (i = 0; i < h; i++)
    {
        for (j = (n - 1); j > 0; j--)
        { 
            printf(" ");
        }
        for (k = 0; k <= i; k++)
        {
            printf("#");
        }
        printf("  ");
        for (k = 0; k <= i; k++)
        {
            printf("#");
        }    
        printf("\n");
        n--;
    }
}    
    

    

File: cs50-2020/pset6/mario/mario.py
from cs50 import get_int

while True:
    h = get_int("Height: ")
    if(h > 0 and h <= 8):
        break
n = h 
y = 0
for i in range(h):
   for j in range(n-1):
       print(' ', end = '')
   n -= 1
   for x in range(y+1):
       print('#', end = '')
   y += 1
   print()
